internal/api/integrations/k8s/k8s.go

```diff
@@ -52,8 +62,27 @@ func (k *K8sClient) ListAll(namespace string) ([]models.Release, error) {
 	for _, r := range releaseList {
 		annotations := helmReleaseAnnotations(r.GetAnnotations())
 
+		gitRepo, err := findVCSRepo(annotations.Code())
+		if err != nil {
+			k.logger.Log("error", err)
+		}
+
+		releaseName := r.GetName()
+
+		image, err := GetImageForRepo(releaseName, r.Spec.Values)
+		if err != nil {
+			k.logger.Log("error", err)
+		}
+
+		repository := ""
```

Comment:
It's personal preference, but explicitly assigning the zero-value smells weird to me in Go. I'd write these like `var repository string`, but there's no significant difference.